srcts/src/bindings/input/selectInput.ts

```diff
@@ -42,12 +43,12 @@ class SelectInputBinding extends InputBinding {
   find(scope: HTMLElement): JQuery<HTMLElement> {
     return $(scope).find("select");
   }
-  getType(el: HTMLElement): string {
+  getType(el: HTMLElement): string | false {
     const $el = $(el);
 
     if (!$el.hasClass("symbol")) {
       // default character type
-      return null;
+      return false;
```

Comment:
Returning `false` instead of `null`; Just need a `false`-y value in the end.

Reply:
Let's use `null`